app/Models/Server.php

```diff
@@ -140,11 +120,11 @@ public static function getUserServers($paginate = null)
             'allocations.port',
             'services.name as a_serviceName',
             'service_options.name as a_serviceOptionName'
-        )->join('nodes', 'servers.node', '=', 'nodes.id')
+        )->join('nodes', 'servers.node_id', '=', 'nodes.id')
         ->join('locations', 'nodes.location', '=', 'locations.id')
-        ->join('services', 'servers.service', '=', 'services.id')
-        ->join('service_options', 'servers.option', '=', 'service_options.id')
-        ->join('allocations', 'servers.allocation', '=', 'allocations.id');
+        ->join('services', 'servers.service_id', '=', 'services.id')
+        ->join('service_options', 'servers.option_id', '=', 'service_options.id')
+        ->join('allocations', 'servers.allocation_id', '=', 'allocations.id');
```

Comment:
Do we need all this information to list user servers? This should be done with the new model stuff instead of manually joining.